trunk/NDHMS/Data_Rec/module_namelist.F

```diff
@@ -55,11 +55,13 @@ subroutine read_rt_nlst(nlst)
           character(len=256) :: route_lake_f=""
           character(len=256) :: route_direction_f=""
           character(len=256) :: route_order_f=""
-          logical            :: reservoir_persistence
+          logical            :: reservoir_persistence_usgs
           character(len=256) :: reservoir_parameter_file=""
           character(len=256) :: reservoir_timeslice_path = ""
           integer            :: reservoir_observation_lookback_hours
           integer            :: reservoir_observation_update_time_interval_seconds
+          logical            :: reservoir_rfc_forecasts
+          logical            :: reservoir_not_levelpool
```

Comment:
I'm a bit out of the loop with respect to this development, but this doesn't appear to be in the documentation and it's not clear to me what it does.

Reply:
reservoir_rfc_forecasts is a boolean flag to turn on/off the RFC Forecasts module. reservoir_not_levelpool is another boolean flag that is set to true if either USGS Persistence, Army Corps Persistence, or RFC Forecasts are set to true which then dictates reservoir_type to be read from a reservoir parameter file. I can add the documentation for these two variables to RFC_Forecasts_Module_README.md right now if you want or add that in the next PR.

Reply:
The name `reservoir_not_levelpool` is awkward; could we rename it something like `reservoir_type_specified` etc. ?

Reply:
Good point. I will rename that now along with document these two and push in a few minutes. @jdmattern-noaa fixing.

Reply:
I pushed these changes.